browser/src/control/Control.JSDialog.js

```diff
@@ -312,6 +312,8 @@ L.Control.JSDialog = L.Control.extend({
 		};
 
 		if (isModalPopup && data.hasClose) {
+			container.style.minWidth = '198px';
```

Comment:
maybe better to keep visual properties in CSS? what do you think?
it already has class: .modalpopup.jsdialog-container so can be targetted
if it's important that it has close button then we can put hasClose class also

Reply:
Yup, I never too sure were to put it. On one hand if it stays in the JS then anyone editing JS files will see it and it will avoid people add in style properties via JS (that override the stuff that is on the CSS side). But I agree I prefer to keep it in the css